File: samples/AdventureWorksLT.WpfApp/TotalDueConditionalFormat.cs
using DevZest.Data;
using System.Windows.Controls;
using DevZest.Data.Presenters;
using System.Windows;

namespace DevZest.Samples.AdventureWorksLT
{
    public sealed class TotalDueConditionalFormat : RowBindingBehavior<TextBlock>
    {
        public TotalDueConditionalFormat(_Decimal totalAmt)
        {
            _totalAmt = totalAmt;
        }

        private _Decimal _totalAmt;

        protected override void Setup(TextBlock view, RowPresenter presenter)
        {
        }

        protected override void Refresh(TextBlock view, RowPresenter presenter)
        {
            view.FontWeight = presenter.GetValue(_totalAmt) >= 10000 ? FontWeights.Bold : FontWeights.Normal;
        }

        protected override void Cleanup(TextBlock view, RowPresenter presenter)
        {
        }
    }
}

File: samples.vb/AdventureWorksLT.WpfApp/TotalDueConditionalFormat.vb
Imports DevZest.Data
Imports DevZest.Data.Presenters

Public NotInheritable Class TotalDueConditionalFormat
    Inherits RowBindingBehavior(Of TextBlock)
    Public Sub New(totalAmt As _Decimal)
        _totalAmt = totalAmt
    End Sub

    Private _totalAmt As _Decimal

    Protected Overrides Sub Setup(view As TextBlock, presenter As RowPresenter)
    End Sub

    Protected Overrides Sub Refresh(view As TextBlock, presenter As RowPresenter)
        view.FontWeight = If(presenter.GetValue(_totalAmt) >= 10000, FontWeights.Bold, FontWeights.Normal)
    End Sub

    Protected Overrides Sub Cleanup(view As TextBlock, presenter As RowPresenter)
    End Sub
End Class